pkg/urbit/include/vere/vere.h

```diff
@@ -1273,6 +1273,10 @@
         void
         u3_daemon_grab(void* vod_p);
 
+
+        c3_w
+    		u3_readdir_r(DIR *dirp, struct dirent *entry, struct dirent **result);
```

Comment:
Please use spaces instead of tabs, and follow the indentation conventions of the surrounding code.

Reply:
done